packages/gatsby/package.json

```diff
@@ -32,6 +32,8 @@
     "babel-plugin-remove-graphql-queries": "^2.0.2-rc.3",
     "better-queue": "^3.8.6",
     "bluebird": "^3.5.0",
+    "cache-manager": "^2.9.0",
+    "cache-manager-fs": "^1.0.6",
```

Comment:
There's also a [redis cache manager](https://www.npmjs.com/package/cache-manager-redis) 👍 

I haven't thought through how to swap these out yet (gatsby-plugin-cache?) but that's something we'll want to think about.